samples/oauth2-security/src/main/java/org/wso2/msf4j/example/Application.java

```diff
@@ -29,6 +29,9 @@
 
     private static final Logger logger = LoggerFactory.getLogger(Application.class);
 
+    private Application() {
+    }
+
```

Comment:
Avoid creating an instance of the Application class from outside the class (there is no reason to do so)